### PR TITLE
Python functionality

### PR DESCRIPTION
**Pull request by yuriimchg**

### CODE REVIEW

File: python_functionality/homework.py [hunk 1]
```diff
@@ -14,7 +14,12 @@ def task_1_fix_names_start_letter(data: DT) -> DT:
         fix_names_start_letters([{'name': 'Alex', 'age': 26}, {'name': 'denys', 'age': 89}])
         >>> [{'name': 'Alex', 'age': 26}, {'name': 'Denys', 'age': 89}]
     """
-    pass
+    for i in data:
+        if i.get('name') is not None and i['name'][0].islower():
+            i['name'] = list(i['name'])
+            i['name'][0] = (i['name'][0].upper())
+            i['name'] = str(''.join(i['name']))
+    return data
```
Comment (dimigor) [inline python_functionality/homework.py]: try to simplify it

File: python_functionality/homework.py [hunk 2]
```diff
@@ -25,7 +30,10 @@ def task_2_remove_dict_fields(data: DT, redundant_keys: List[str]) -> DT:
        remove_dict_field([{'name': 'Alex', 'age': 26}, {'name': 'denys', 'age': 89}], 'age')
         >>> [{'name': 'Alex'}, {'name': 'denys'}]
     """
-    pass
+    for data_list in data:
+        for redun_list in redundant_keys:
+            data_list.pop(redun_list)
```
Comment (dimigor) [inline python_functionality/homework.py]: what about dict comprehensions?

**Review by dimigor — CHANGES_REQUESTED** [review]

File: python_functionality/homework.py (hunk 1 above)
Comment (AlexandrVovk) [reply python_functionality/homework.py]: done

File: python_functionality/homework.py (hunk 2 above)
Comment (AlexandrVovk) [reply python_functionality/homework.py]: it was hard, but I did it

**Review by dimigor — APPROVED** [review]

**Pull request merged**